index.html

```diff
@@ -26,15 +26,17 @@ <h3>Maturity</h3>
 
 <p>Note that we do sometimes swear like sailors on comms. Profanity is fine, slurs are not.</p>
 
+<p>While it’s (unlike in real life) fine to drink and fly or "fly high", please bear in mind that you’re always expected to abide by the white paper, and that how drunk or high you are is probably a lot less interesting to your corpmates than it seems to you. Also, while it’s fine to talk about sexual topics on Teamspeak, please respect that your alliancemates might not want to hear about it - and if someone asks you to take the discussion somewhere else, do so without arguing about it.</p>
+
 <p>Also note that some phrases, behaviors, and topics are already known to make members uncomfortable and are to be avoided:</p>
 
 <ul>
 <li>Uses of 'gay' and similar as synonyms for 'uncool';</li>
 <li>Uses of 'jew' and similar as synonyms for 'carebear';</li>
 <li>Uses of 'bitch' and similar for any reason;</li>
 <li>Uses of 'rape' and related verbs/terms for any reason;</li>
-<li>Discussions of drug use or sex</li>
-<li>Nationalism/racism of either the 'USA #1!' or 'russians are all drunk' types</li>
+<li>Nationalism/racism of either the 'USA #1!' or 'so-and-so race are all drunk'
+types, even if you're being ironic</li>
```

Comment:
Being ironic is more generally not acceptable for any of these behaviors. Maybe this can be factored out?

Reply:
Done!